README.md

```diff
@@ -1,6 +1,6 @@
 Authors Haven - A Social platform for the creative at heart.
 =======
-
+[![Build Status](https://travis-ci.com/andela/ah-bird-box.svg?branch=ch-integrate-travisCI-163388242)](https://travis-ci.com/andela/ah-bird-box)
```

Comment:
Hi good work, but I think it would be better if the badge pointed to develop branch.